cli/config.py

```diff
@@ -123,14 +123,14 @@ def name_prefix(self, value: str) -> None:
         self._config['name_prefix'] = value
 
     @property
-    def enable_carbon_black_downloader(self) -> int:
+    def enable_carbon_black_downloader(self) -> bool:
         return self._config['enable_carbon_black_downloader']
 
     @enable_carbon_black_downloader.setter
-    def enable_carbon_black_downloader(self, value: int) -> None:
-        if value not in {0, 1}:
+    def enable_carbon_black_downloader(self, value: bool) -> None:
+        if not isinstance(value, bool):
```

Comment:
Do you consider backward compatible? I would also convert `0/1` to `false/true` here.

Reply:
The config is already changing, so I'm not really worried about the backwards compatibility. I did convert it to `true/false` (Which is why it's now checking for a `bool`)